functions/src/google/test.json

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
Does this get used? I could not find `test.json` in the PR

Reply:
nope this was something I noticed too I have some clean up :)